qa/evil-tests/src/test/java/org/elasticsearch/tribe/TribeUnitTests.java

```diff
@@ -59,8 +60,24 @@ public static void createTribes() {
             .put("node.mode", NODE_MODE)
             .put("path.home", createTempDir()).build();
 
-        tribe1 = NodeBuilder.nodeBuilder().settings(Settings.builder().put(baseSettings).put("cluster.name", "tribe1").put("node.name", "tribe1_node")).node();
-        tribe2 = NodeBuilder.nodeBuilder().settings(Settings.builder().put(baseSettings).put("cluster.name", "tribe2").put("node.name", "tribe2_node")).node();
+        tribe1 = NodeBuilder
+            .nodeBuilder()
+            .settings(
+                Settings.builder()
+                    .put(baseSettings)
+                    .put("cluster.name", "tribe1")
+                    .put("node.name", "tribe1_node")
+                    .put(DiscoveryService.SETTING_DISCOVERY_SEED, random().nextLong())
```

Comment:
> does this mean that users will have to specify the discovery seed in their configuration for each tribe?

No. It is just needed for this test otherwise all the generated nodes will have the same seed and so the same node id and the test will fail.